lib/assert.js

```diff
@@ -31,6 +31,8 @@ const { inspect } = require('util');
 
 const assert = module.exports = ok;
 
+const NO_EXCEPTION_SENTINEL = {};
```

Comment:
Would a symbol work here as well?

Reply:
It's merged now but for posterity: yes, but it doesn't add anything extra and it uses more memory than a plain empty object literal.